CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# Polly.Caching.IDistributedCache change log
+# Polly.Caching.Distributed change log
+## 1.0.0
```

Comment:
Need to make this

    ## 2.0.0 

as a version 1.0.x of this package has already been released.